source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/overview/reviewers/form.tsx

```diff
@@ -50,6 +57,19 @@ class AddReviewerForm extends React.Component<IProps> {
     });
   };
 
+  setEmptyLocale = () => {
+    const { currentUser, i18n } = this.props;
+    const { locale } = this.state;
+    var defaultLocale = locale;
+    if (isEmpty(locale)) {
+      const attributes = attributesFor(currentUser) as UserAttributes;
+      const userLocale = attributes.locale;
```

Comment:
this could be simplified to:
`const { locale: userLocale } = attributesFor(...`